### PR TITLE
LLSE: Some PlayerAPI properties and `setFire`

### PR DESCRIPTION
**Pull request by ghost**
## Description

Please carefully read the [Contributing note](https://github.com/LiteLDev/LiteLoaderBDS/discussions/546) before making any pull requests.
And, **Do not make a pull request to merge into main branch unless it is a hotfix. Use the beta branch instead.**
## Changelog

- **PlayerAPI**
    - Add `canFly` parameter.
    - Add `canSleep` parameter.
    - Function `setOnFire` is now `setFire`, also added `isEffect` argument

## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [x] Documentation

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] My pull request is unique and no other pull requests have been opened for these changes
- [X] I have read the [Contributing note](https://github.com/LiteLDev/LiteLoaderBDS/discussions/546)
- [X] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: docs/zh_CN/Development/GameAPI/Player.md [hunk 1]
```diff
@@ -52,6 +52,8 @@
 | pl.uuid      | 玩家Uuid字符串              | `String`         |
 | pl.permLevel | 玩家的操作权限等级（0 - 4） | `Integer`        |
 | pl.gameMode  | 玩家的游戏模式（0 - 3）     | `Integer`        |
+| pl.canFly    | 找出玩家是否可以飞行        | `Boolean`        |
```
Comment (Jaciezyt) [inline docs/zh_CN/Development/GameAPI/Player.md]:
```suggestion
| pl.canFly    | 玩家是否可以飞行        | `Boolean`        |
```

File: docs/zh_CN/Development/GameAPI/Player.md [hunk 2]
```diff
@@ -52,6 +52,8 @@
...
+| pl.canSleep  | 玩家可以睡觉吗              | `Boolean`        |
```
Comment (Jaciezyt) [inline docs/zh_CN/Development/GameAPI/Player.md]:
```suggestion
| pl.canSleep  | 玩家是否可以睡觉              | `Boolean`        |
```

File: ScriptEngine/API/PlayerAPI.cpp [hunk 3]
```diff
@@ -1526,6 +1551,25 @@ Local<Value> PlayerClass::hurt(const Arguments& args) {
...
+        int time = args[0].toInt();
+        bool isEffectValue = args[1].asBoolean().value();
+
+        bool result = player->setOnFire(time, isEffectValue);
+        return Boolean::newBoolean(result);
+    }
+    CATCH("Fail in setOnFire!");
+}
+
 Local<Value> PlayerClass::setOnFire(const Arguments& args) {
```
Comment (Jaciezyt) [inline ScriptEngine/API/PlayerAPI.cpp:1574]: Add `For compatibility` comment

**Review by Jaciezyt — CHANGES_REQUESTED** [review]

File: ScriptEngine/API/PlayerAPI.cpp (hunk 3 above)
Comment (ghost) [reply ScriptEngine/API/PlayerAPI.cpp:1574]: > Add `For compatibility` comment

Added!

**Comment by Jaciezyt** [comment]
Thank you for your time and this contribution! 👍 😄

**Review by Jaciezyt — APPROVED** [review]

**Pull request merged**